src/qibolab/instruments/qm.py

```diff
@@ -164,10 +164,10 @@ def connect(self):
         host, port = self.address.split(":")
         self.manager = QuantumMachinesManager(host, int(port))
 
-    def setup(self, qubits, relaxation_time=0, time_of_flight=0, smearing=0):
-        self.time_of_flight = time_of_flight
-        self.smearing = smearing
-        self.relaxation_time = relaxation_time
+    def setup(self, qubits, *args, **kwargs):
+        self.time_of_flight = kwargs.get("time_of_flight", 0)
+        self.smearing = kwargs.get("smearing", 0)
+        self.relaxation_time = kwargs.get("relaxation_time", 50000)
```

Comment:
You can keep the arguments spelled out as before, even if the abstract interface uses `*args, **kwargs`, since it is just a specialization.

Reply:
The issue I had here was that this function is called by the platform [here](https://github.com/qiboteam/qibolab/blob/fa1ce933c06cc21e335efcc43c415b02bbfd85a2/src/qibolab/platforms/platform.py#L30) and if `**options` contains additional keywords that are not spelled out it fails.

Reply:
But, to solve this issue, you can just add a `**_kwargs` argument.
```suggestion
    def setup(self, qubits, relaxation_time=0, time_of_flight=0, smearing=0, **_kwargs):
        self.time_of_flight = time_of_flight
        self.smearing = smearing
        self.relaxation_time = relaxation_time
```

If you prepend an underscore, this will also encode the intention of not using it.